### DISCUSSION TITLE
TODO list

### DISCUSSION THREAD

**Discussion by luc-github** [luc-github/ESP3DLib#1] [General]
Short term:
* Import embedded page code (🆗)
* Add documentation / populate wiki (🆗 )
* Add issue templates, labels list, stale configuration (🆗 )

Mid term:
* Fix issues with current code:
    - Use  WIFI_SSID and WIFI_PWD from Marlin (🆗 )
    - allow to configure all ESP3D settings from Configuration_adv.h (🆗 )
    - TBD
* Import ESP3D v2.1 features (TBD)
* move to ESP3D v3.X (TBD)

**Comment by luc-github** [comment]
In coming month I will focus on ESP3D-WEB UI for version 3.0 as it is waiting for 1 year already

so import of ESP3D v2.1 features may be dropped unless there are urgent features that could be easily imported, TBD
 
Feel free to list here the priority needs/expectations ^_^

**Comment by Tigger777** [comment]
My 2c worth:
- More macro buttons and less the CNC style control in the command panel.
- Static buttons for emergency parser commands. To make pausing prints and colour changes easier and temperature changes during waits possible.
- MQTT support for tasmota style wifi sockets to drop power after print completes at a minimum. It would be nice to have a time-lapse camera trigger from gcode via MQTT as well.
